modules/wallet/persist.go

```diff
@@ -108,12 +90,39 @@ func (w *Wallet) load() error {
 		}
 	}
 
-	// Load the siafunds file, which is intentionally called 'outputs.dat'.
-	var siafundAddresses []types.UnlockHash
-	err = encoding.ReadFile(filepath.Join(w.saveDir, "outputs.dat"), &siafundAddresses)
+	return nil
+}
+
+// load reads the contents of a wallet from a file.
+func (w *Wallet) load() error {
+	var savedKeys []savedKey
+	err := encoding.ReadFile(filepath.Join(persist.HomeFolder, modules.WalletDir, "wallet.dat"), &savedKeys)
```

Comment:
I think the next iteration of the wallet (which will introduce 'accounts' -> one file per account) will solve a lot of these problems.